test/unitTests/observers/telemetryObserver.test.ts

```diff
@@ -226,4 +226,14 @@ describe('TelemetryObserver', () => {
             });
         });
     });
+
+    it(`LearnMoreClick: 'LearnMore.Click' event should be sent`, () => {
```

Comment:
feel like we have too many this kind of tests, which are testing the same thing.

Reply:
they are testing the different function

Reply:
they are testing the different function, I keep it because it truly helps to explore some code issue

---
In reply to: [382360670](https://github.com/docascode/vscode-docs-build/pull/45#discussion_r382360670) [](ancestors = 382360670)